rest/admin_api.go

```diff
@@ -409,6 +414,29 @@ func (h *handler) handleGetConfig() error {
 			}
 		}
 
+		// grab cluster uuid for runtime config
+		var clusterUUID string
+		if len(allDbNames) > 0 {
+			// we can use db context to retrieve clusterUUID
+			dbCtx := h.server.Database(h.ctx(), allDbNames[0])
+			clusterUUID = dbCtx.ServerUUID
+		} else {
+			bucketCfg := getBucketConfigFromBoostrap(h.server.Config.Bootstrap)
+			spec, err := GetBucketSpec(h.ctx(), &DatabaseConfig{
+				DbConfig: DbConfig{
+					BucketConfig: bucketCfg,
+				},
+			}, h.server.Config)
+			if err != nil {
+				return err
+			}
+			clusterUUID, err = h.server.BootstrapContext.Connection.GetClusterUUID(h.ctx(), spec)
```

Comment:
I think if persistent config is diable that this bootstrap connection might not exist?